test/kpod_stop.bats

```diff
@@ -6,6 +6,10 @@ function teardown() {
     cleanup_test
 }
 
+function setup() {
+    copy_images
```

Comment:
We only use `redis:alpine` in this file, so I don't think this'll save us any time.